src/presentation/test/integration/registration/name.test.ts

```diff
@@ -62,4 +62,25 @@ describe("Name Page", () => {
     expect(res.text).not.toContain("WELSH -");
   });
 
+  it("should load the private name page with Welsh text", async () => {
```

Comment:
the name of the tests reference the private name page instead of the Scottish name page

Reply:
tests now referencing Scottish name page